packages/repository-json-schema/src/build-schema.ts

```diff
@@ -307,6 +313,37 @@ function getTitleSuffix<T extends object>(options: JsonSchemaOptions<T> = {}) {
   return suffix;
 }
 
+function stringifyOptions(modelSettings: object = {}) {
+  return stringifyObject(modelSettings, {
+    indent: '  ', // two spaces
```

Comment:
Is this setting needed when `inlineCharacterLimit` is set to `Infinity`? I mean there should be never any line breaks, therefore there is nothing to indent.

Thoughts?

Reply:
Your right, in this case is not really needed. Pushed a version without it.